grails-app/services/au/org/ala/bie/ImportService.groovy

```diff
@@ -497,32 +497,29 @@ class ImportService implements GrailsConfigurationAware {
         }
```

Comment:
Knowledgebase needs to be included in the steps in importAll() at ~L100 and in default steps in application.yml

Reply:
Oops forgot that, thanks - will add.